### DISCUSSION TITLE
RISC-V status

### DISCUSSION THREAD

**Discussion by jwinarske** [maplibre/maplibre-native#2704] [Show and tell]
#### Placeholder for RISC-V status

I can successfully build main tip of tree for RISC-V Linux using:
```
-DMLN_WITH_VULKAN=ON
-DMLN_WITH_X11=OFF
-DMLN_WITH_WAYLAND=OFF
-DMLN_DRAWABLE_RENDERER=ON
-DMLN_LEGACY_RENDERER=OFF
-DMLN_WITH_VULKAN=ON
-DMLN_WITH_OPENGL=OFF
-DMLN_WITH_EGL=OFF
```

Target device is a DeepComputing DC-ROMA II Laptop running Ubuntu 23.

**Comment by louwers** [comment]
Very interesting @jwinarske! I transferred this to a discussion, because we use issues for stuff that is actionable.

Could you share some more details of what you are working on?

**Comment by jwinarske** [comment]
@louwers 

This stack will be implemented in OS images for range of platforms

Software Stack
* Flutter Navigation app
* FFI shim module
* maplibre-native module
* OS - Linux - AGL Ricefish
* Cloud - tile server (tbd)
* Cloud - Overture parquet

Target architectures
* x86_64
* aarch64
* riscv64

Audience
OEM
TIER1

Images run with QEMU - arm64 Mac, x86_64 Linux

**Reply by louwers** [reply]
Do you still want to push forward a C FFI? https://github.com/maplibre/maplibre-native/pull/1254

Having this part of the library will make sure it's well tested and remains stable.

**Reply by jwinarske** [reply]
Yes.  For Flutter it's a requirement.
